File: OrchidCore/src/main/java/com/eden/orchid/impl/themes/DefaultTheme.java
package com.eden.orchid.impl.themes;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.theme.Theme;

import javax.inject.Inject;

public class DefaultTheme extends Theme {

    @Inject
    public DefaultTheme(OrchidContext context) {
        super(context, "Default", 100);
    }

}

File: orchid-core/src/main/kotlin/com/eden/orchid/impl/themes/DefaultTheme.kt
package com.eden.orchid.impl.themes

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.options.annotations.Description
import com.eden.orchid.api.theme.Theme

import javax.inject.Inject

@Description(
    value = "The default theme, which adds no assets and lets you build your theme entirely by custom templates.",
    name = "Default"
)
class DefaultTheme @Inject
constructor(context: OrchidContext) : Theme(context, "Default")
